app/src/main/java/timur/gilfanov/messenger/ui/screen/user/ProfileEditUiState.kt

```diff
@@ -0,0 +1,14 @@
+package timur.gilfanov.messenger.ui.screen.user
+
+import android.net.Uri
+import androidx.compose.foundation.text.input.TextFieldState
+import timur.gilfanov.messenger.domain.entity.message.validation.TextValidationError
+
+data class ProfileEditUiState(
+    val nameInput: TextFieldState,
+    val nameInputValidationError: TextValidationError? = null,
+    val picture: Uri,
```

Comment:
Can be no picture in profile:
```suggestion
    val picture: Uri?,
```